earth_enterprise/src/fusion/autoingest/sysman/AssetHandleD.h

```diff
@@ -189,7 +189,7 @@ class MutableAssetHandleD_ : public virtual Base_ {
       uint32 numDotNew=0;
       for (std::vector<std::string>::iterator it = toDelete.begin();
            it != toDelete.end(); ++it) {
-        if (dirtyMap.find(*it) != dirtyMap.end()) {
+        if (dirtyMap.contains(*it)) {
```

Comment:
This function was removed in the master branch. You could remove it here as well to make your work easier.